infrastructure/repository/authzed/SpiceDbAccessRepository.go

```diff
@@ -331,7 +331,7 @@ func (s *SpiceDbAccessRepository) NewConnection(spiceDbEndpoint string, token st
 	if !useTLS {
 		opts = append(opts, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	} else { // Use TLS - but for now SKIP CA verification
```

Comment:
Update the comment to relfect the chnges

Reply:
Ah, while trying to keep it as minimal as possible, I skipped right over that. I missed that the variable was still called skipCA too. Updated both.